docs/src/Tutorials/Message Passing.md

```diff
@@ -406,7 +406,7 @@ julia> Gallager_B(H, y)
 ```
 
 ## Decimation
-Decimation was previously used in message-passing applications outside of error correction and was applied to stabilizer codes in [](@cite). The idea is to freeze the value of a variable node. We can either do this from the start to obtain a so-called *genie-aided* decoder, or we can periodically pause message passing to fix a bit. In *guided decimation*, we pause every fixed number of rounds and freeze the value of the variable node with the highest log-likelihood ratio. In *automated decimation*, we pause after every iteration and fix any bit whose absolute value has passed a certain threshold.
```

Comment:
https://ieeexplore.ieee.org/abstract/document/10619083

Reply:
Thank you!